scripts/lb-wrapper

```diff
@@ -48,9 +48,10 @@ fi
 # 2024-01-21: Not needed as XKLB reports its own version# to /var/log/xklb.log
 # log "Info" "xklb version: $(${XKLB_EXECUTABLE} --version)"
 
-if mv ${SURVEY_DB_FILE} ${SURVEY_DB_FILE}.$(date +%F_%T_%Z) 2> /dev/null; then
-    log "Info" "Old ${SURVEY_DB_FILE} moved aside."
-fi
+# 2024-01-24: Not needed as we don't discard the database anymore
```

Comment:
I would remove dead code instead of commenting it because it's pretty easy to find it again with `git show` or `git log -p -S 'moved aside'`

Reply:
Right.

And blame me!  As I'm "Old Skool" and recommend explanatory comments (within limits!) to foster approachable code + open community 😸

Reply:
Done:

- https://github.com/iiab/calibre-web/pull/109/commits/f9a9084aeae93aebedf9a94b72d8a9699b059b7f